build.psm1

```diff
@@ -136,6 +148,13 @@ function Start-ScriptAnalyzerBuild
             return
         }
 
+        if (-not $profilesCopied)
+        {
+            Copy-CompatibilityProfiles
+            # Set the variable in the caller's scope, so this will only happen once
+            Set-Variable -Name profilesCopied -Value $true -Scope 1
```

Comment:
This is a bit nasty and probably also slows down the execution (since some optimisations have to be turned off), but I'm not entirely sure what a better technique is. `$script:profilesCopied` would be too permanent, but local scope means repeating it for every build target.